docs/src/contributing.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+This page details the some of the guidelines that should be followed when contributing to this package.
+
+### Reporting issues & discussing
+
+1. If you need explanation on how to do X, Y using Distributions,
+feel free to ask on the Julia [Discourse](https://discourse.julialang.org/c/domain/stats)
+or [Slack](https://julialang.slack.com), get an invitation
+[here](https://slackinvite.julialang.org/).
+
+2. If you have a bug linked with Distributions, check that it has
+not been reported yet on the issues of the repository.
+If not, you can file a new issue, add your version of the package
+which you can get with this command in the Julia REPL:
+```julia
+julia> ]status Distributions
+```
+
+Be exhaustive in your report, give the summary of the bug,
+a Minimal Working Example (MWE), what happens and what you
+expected to happen.
+
+### Workflow with Git and GitHub
+
+To contribute to the package, fork the repository on GitHub,
+clone it and make modifications on a new branch,
+**do not commit modifications on master**.
+Once your changes are made, push them on your fork and create the
+Pull Request on the main repository.
+
+### Requirements
+
+Distributions is a central package which many rely on,
+the following are required for contributions to be accepted:
+1. Docstrings must be added to all interface and non-trivial functions.
+2. Tests validating the modified behavior in the `test` folder. If new test files are added, do not forget to add them in `test/runtests.jl`. Cover possible edge cases. Run the tests locally before submitting the PR.
+3. At the end of the tests, `Test.detect_ambiguities(Distributions)` is run to check method ambiguities. Verify that your modified code did not yield method ambiguities.
+4. Make according modifications to the `docs` folder, build the documentation locally with `$ julia docs/make.jl`, verify that your modifications display correctly and did not yield warnings.
+
+## Branches
+
+This package uses `release-*` branches for tagged minor versions. This follows the same approach used in the main Julia repository, albeit on a much more modest scale.
+
+Please open pull requests against the `master` branch rather than any of the `release-*` branches whenever possible.
+
+### Backports
+
+Bug fixes are backported to the `release-*` branches using `git cherry-pick -x` and will become available in point releases of that particular minor version of the package.
+
+Feel free to nominate commits that should be backported by opening an issue. Requests for new point releases to be tagged in `METADATA.jl` can also be made in the same way.
+
+### `release-*` branches
```

Comment:
We don't have release branches

Reply:
Hmm, on reflection and further reading: 
What to make of the `/^v\d+\.\d+(\.\d+)?(-\S*)?$/` filter that was just added to Travis? Should this section be reworded to refer to the Travis regex naming convention? Or removed entirely?

Reply:
It can be removed. The travis regex is just declaring to push to documentation major and minor releases